examples/zipkin-tracing/front-envoy-zipkin.json

```diff
@@ -9,7 +9,7 @@
           "config": {
             "generate_request_id": true,
             "tracing": {
-              "operation_name": "ingress"
+              "operation_name": "egress"
             }, 
```

Comment:
shouldn't this be ingress? This is for front envoy (user facing).. on port 80.

Reply:
Prior to this change, the front-envoy generated a client span which indicates an outbound request - which is egress.

I guess I view the front-envoy as being a proxy for the user (i.e. an api gateway) - so in that respect it is representing the egress from that user app.

If we wanted the front-envoy to be reflected as a separate (api gateway) service in its own right, then possibly we need a different operation name that represents both the receiving of the request (ingress) and the routing of that request to the actual service (egress) - but from the "application" level I think this front-envoy is just acting as a 'proxy' for the user app.

Reply:
Maybe the solution would be to move away from the ingress/egress terms, to instead use more tracing terminology - i.e. "span_kind": "client" (or "server") - but there would need to be a transition period where the span kind would need to be inferred from the "operation_name".